worlds/hospital.world

```diff
@@ -1069,599 +1074,5 @@
                 <falloff>1</falloff>
```

Comment:
Does ceiling_light_0 also needs to be removed? or is it intentional?

Reply:
I don’t know why, but thought it would be good to have a single light - as most worlds have a couple defined. Not a requirement though!